src/xcode/ENA/ENA/Source/Scenes/QRScanner/QRScannerViewController.swift

```diff
@@ -145,7 +148,7 @@ class QRScannerViewController: UIViewController {
 		instructionDescription.textAlignment = .center
 		instructionDescription.textColor = .enaColor(for: .iconWithText)
 		instructionDescription.font = .enaFont(for: .body)
-		instructionDescription.text = AppStrings.UniversalQRScanner.instructionDescription
+		instructionDescription.text = (presenter != nil && presenter == .onBehalfFlow) ? AppStrings.UniversalQRScanner.instructionDescriptionWarnOthers : AppStrings.UniversalQRScanner.instructionDescription
```

Comment:
Optional: to keep line good readable, you could use a private calc property like

```swift
instructionDescription.text = instructionDescription

...

private var instructionDescription: String {
   guard isPresenterOnBehalfFlow else { 
      return AppStrings.UniversalQRScanner.instructionDescription 
   }
   return AppStrings.UniversalQRScanner.instructionDescriptionWarnOthers
}
```

Reply:
Thanks Felix!
After discussed it with Naveed I did it in this way.